azure-pipelines.yml

```diff
@@ -100,7 +100,7 @@ stages:
         chmod +x ./DylibRepack-arm
         mv ./DylibRepack-arm ./DylibRepack
         mkdir ${BUILD_SOURCESDIRECTORY}/nuget/arm/repack
-        ./DylibRepack repack -o ${BUILD_SOURCESDIRECTORY}/nuget/arm/repack ${BUILD_SOURCESDIRECTORY}/nuget/arm/libOpenCvSharpExtern.dylib /opt/homebrew/lib/libgdiplus.dylib 
+        ./DylibRepack repack -o ${BUILD_SOURCESDIRECTORY}/nuget/arm/repack ${BUILD_SOURCESDIRECTORY}/nuget/arm/libOpenCvSharpExtern.dylib /opt/homebrew/lib/libgdiplus.dylib /opt/homebrew/lib/libpng.dylib 
```

Comment:
Why is this needed? OpenCV builds its own version of the libpng library. We should use that instead of using the installed version.